package.json

```diff
@@ -43,14 +43,14 @@
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^8.0.1",
-    "jest": "^28.1.3",
+    "jest": "^29.0.0",
```

Comment:
Relates to https://github.com/jestjs/jest/issues/14741#issuecomment-1845219606 otherwise I cannot run tests locally